app-frontend/src/app/services/map/map.service.js

```diff
@@ -227,6 +233,36 @@ class MapWrapper {
         return this;
     }
 
+    rectifyCoords(latlongs) {
+        return latlongs.map((multipolygon) => {
+            return multipolygon.map((polygon) => {
+                return polygon.map((cord) => {
+                    return {lng: cord.lng > 0 ? cord.lng - 360 : cord.lng, lat: cord.lat};
+                });
+            });
+        });
+    }
+
+    shouldRectifyBounds(bounds, beforeArea, afterArea) {
+        return bounds &&
+              Math.abs(bounds.getEast() + bounds.getWest()) < 30 &&
```

Comment:
30 is the degree threshold? Magic number could use an explanation (and comparison to some reference values for MODIS e.g.)

Reply:
hm yea it is. I'll do that.